### DISCUSSION TITLE
The Blast Library still refer to PhysX4.1,not PhysX5?

### DISCUSSION THREAD

**Discussion by Artisic** [NVIDIA-Omniverse/PhysX#66] [Q&A]
I want to Use Blast With PhysX5 and find that it still refer to PhysX4.1.
Is there anyway possible that I can use Blast with PhysX5 ?

**Comment by nvidia-bgaldrikian** [comment]
Hi Artisic,

The PhysX integration in the Blast SDK should be encapsulated within the ExtPx extension (and related ExtPx serialization extension), which is currently not part of the open source release.  However, there was a documentation error, with parts that still referred to ExtPx.  We will be updating the open source repo to include ExtPx and it will be updated to use PhysX5.  My apologies for the delay in getting those extensions out.

Bryan Galdrikian

**Reply by nvidia-bgaldrikian** [reply]
Update:
We've gone a different route with this, which I think is the correct route - everything except the ExtPx extension and Px serialization code has been made completely physx-independent.  This required bringing in replacements for some Px types, which we've done (there are Nv types such as NvVec3 used by the Tk library API, and several Nv types used in the underlying source).  This will be pushed to the open source repo in late February.  We still plan to enable the ExtPx extension, and I hope to make it easier to use the physx version & build of your choice with that extension.

**Comment by nvidia-bgaldrikian** [comment]
Hi @Artisic, just wanted to give you a heads-up that the changes I described a few weeks ago are now pushed up to the public repo.  Hopefully this will work painlessly for you!
